### PR TITLE
Add Seeed_XIAO_nRF52840_Sense board info

### PR DESCRIPTION
**Pull request by 0hotpotman0**

### CODE REVIEW

**Review by makermelissa — CHANGES_REQUESTED** [review]
Hi, thanks for submitting. Please keep the features limited to the list on this page: https://learn.adafruit.com/how-to-add-a-new-board-to-the-circuitpython-org-website/adding-to-downloads. You can move the text for the feature list you do have down into the description.

The reason for that is the features filter options are automatically generated based on the items and even slight spelling differences will generate duplicate options. Thanks.

**Comment by 0hotpotman0** [comment]
> Hi, thanks for submitting. Please keep the features limited to the list on this page: https://learn.adafruit.com/how-to-add-a-new-board-to-the-circuitpython-org-website/adding-to-downloads. You can move the text for the feature list you do have down into the description.
> 
> The reason for that is the features filter options are automatically generated based on the items and even slight spelling differences will generate duplicate options. Thanks.

Make sense

File: _board/Seeed_XIAO_nRF52840_Sense.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+Seeed XIAO BLE Sense contains a wealthy interface. The first thing to note is that the Near Field Communication(NFC) is functional on the board. Secondly, there is a tiny and elegant reset button on one side of the Type-C interface. On the other side, it is designed in a three-in-one LED along with a power LED. There are 11 digital i/o that can be used as PWM pins and 6 analog i/o that can be used as ADC pins. It supports UART, IIC, and SPI all three common serial ports. Same as Seeed XIAO RP2040, it has an onboard 2 MB flash which means it can also be programmed by Arduino, MicroPython, CircuitPython, or other program languages. 
+
+
+This board hasn't been fully documented yet. Please make a pull request adding more info to this file.
+
+The description should be written to inform a CircuitPython user what makes the board unique and link to relevant info about it.
```
Comment (makermelissa) [inline _board/Seeed_XIAO_nRF52840_Sense.md:27]: I'm guessing you didn't mean to leave these lines.

**Review by makermelissa — CHANGES_REQUESTED** [review]
Thanks, looks like there's still some default text.

File: _board/Seeed_XIAO_nRF52840_Sense.md (hunk 1 above)
Comment (0hotpotman0) [reply _board/Seeed_XIAO_nRF52840_Sense.md]: Oops... I didn't realize it...

**Review by makermelissa — APPROVED** [review]
Thanks

**Pull request merged**